app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(team: list) -> int:
+    return sum(player.get_rating() for player in team)
+
+
+def elves_concert(elves: str) -> None:
```

Comment:
You need list not str annotation in elves concert function. You can annotate like list[Elves] just import classes for such action.